primer/brand
# [MinimalFooter] Fixes link and button alignment

## Summary

Adjust alignment when on MinimalFooter when using links and button.


## List of notable changes:

- **added** story with mixed buttons and links
- **updated** MinimalFooter styles and Stack alignment


## What should reviewers focus on?

- Storybook Links&Buttons:

## Steps to test:

1. Open the # component in CI-deployed preview environment
2. Go to # story in Storybook
3. Verify that links-and buttons are vertically aligned


## Supporting resources (related issues, external links, etc):

- Fixes: https://github.com/primer/brand/issues/286

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">
<img src="https://github.com/primer/brand/assets/912236/58ff2d4a-c304-46f2-9654-9373b3cd39a4">

 </td>
<td valign="top">
<img src="https://github.com/primer/brand/assets/912236/21e1de76-72b4-4cac-978a-bbe8b376815f">

</td>
</tr>
</table>
